.coveragerc

```diff
@@ -0,0 +1,6 @@
+[run]
+omit = 
+    # omit everything in the tests directory
+    tests/*
+    # omit the virtual environment, don't forget to change the name of your venv in place of venv1
+    venv1/*
```

Comment:
I don't think that we need it. We don't make a virtualenv on Github CI while testing, therefore it can be ignored

Reply:
@devkapilbansal you mean to say only the virtual environment part right? If so I will remove it

Reply:
Yes only virtualenv